workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -60,14 +67,18 @@ process salmon_index_full_sa{
       -d decoys.txt \
       -i ${index_base}_k${kmer}_full_sa \
       -k ${kmer} \
-      -p ${task.cpus}
+      -p ${task.cpus} \
+      --gencode
     """
 }
 
 process kallisto_index{
   container 'quay.io/biocontainers/kallisto:0.46.2--h4f7b962_1'
   publishDir "${params.ref_dir}/kallisto_index", mode: 'copy'
-  memory 32.GB
+  memory { 120.GB * task.attempt}
+  cpus 8
```

Comment:
Just to note that these are probably redundant here: once you get to a 120GB memory machine, it already has at least 16 cores, based on https://aws.amazon.com/ec2/instance-types/ (I think it will choose an R5a machine when it needs a lot of memory.)